### ISSUE TITLE
The content of the TabStrip's tabs is not focusable

### ISSUE THREAD

**Issue by IvanDanchev** [telerik/kendo-ui-core#5871]
### Bug report
Reported in Ticket ID: 1472894.

### Reproduction of the problem
Reproducible in this [demo](https://demos.telerik.com/aspnet-mvc/tabstrip/keyboard-navigation).

### Current behavior
The content of the tab is not focusable. 

### Expected/desired behavior
The tabs themselves and the active tab's content should be focusable: [https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html](https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html)

### Environment

* **Kendo UI version:** 2020.2.617
* **jQuery version:** x.y
* **Browser:** [all ]
